packages/signature-controller/src/SignatureController.ts

```diff
@@ -902,6 +914,13 @@ export class SignatureController extends BaseController<
     request: OriginalRequest,
     chainId: string,
   ) {
+    if (
+      !this.#getFeatureFlags ||
+      this.#getFeatureFlags().disableDecodingApi ||
```

Comment:
Is decoding enabled or disabled by default?

Currently decoding is disabled if `getFeatureFlags` isn't provided?

Reply:
`getFeatureFlags` is important as we need to know if user has these preferences correctly set before invoking api